.github/workflows/linux_riscv_xuantie.yml

```diff
@@ -0,0 +1,256 @@
+    steps:
+      - name: Checkout
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        timeout-minutes: 15
+
+      - uses: ./.github/actions/handle_docker
+        id: handle_docker
+        with:
+          images: |
+            ov_build/ubuntu_22_04_riscv_xuantie
+            ov_test/ubuntu_22_04_riscv_xuantie
+          registry: 'openvinogithubactions.azurecr.io'
+          dockerfiles_root_dir: '.github/dockerfiles'
+          changed_components: ${{ needs.smart_ci.outputs.changed_components }}
+
+  Build:
+    needs: [Smart_CI, Docker]
+    timeout-minutes: 150
+    defaults:
+      run:
+        shell: bash
+    runs-on: aks-linux-16-cores-32gb
+    container:
+      image: ${{ fromJSON(needs.docker.outputs.images).ov_build.ubuntu_22_04_riscv_xuantie }}
+      volumes:
+        - /mount:/mount
+        - ${{ github.workspace }}:${{ github.workspace }}
+    env:
+      CMAKE_BUILD_TYPE: 'Release'
+      CMAKE_GENERATOR: 'Ninja'
+      CMAKE_CXX_COMPILER_LAUNCHER: ccache
+      CMAKE_C_COMPILER_LAUNCHER: ccache
+      OPENVINO_REPO: ${{ github.workspace }}/openvino
+      BUILD_DIR: ${{ github.workspace }}/build
+      INSTALL_DIR: ${{ github.workspace }}/openvino_install
+      INSTALL_TEST_DIR: ${{ github.workspace }}/tests_install
+      CCACHE_REMOTE_DIR: /mount/caches/ccache/ubuntu22_riscv64/xuantie/${{ needs.Smart_CI.outputs.target_branch }}
+      CCACHE_DIR: ${{ github.workspace }}/ccache
+      CCACHE_TEMPDIR: ${{ github.workspace }}/ccache_temp
+      CCACHE_MAXSIZE: 2G
+
+    if: ${{ github.event_name != 'merge_group' }}
+    steps:
+      - name: Clone OpenVINO
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        timeout-minutes: 15
+        with:
+          path: ${{ env.OPENVINO_REPO }}
+          submodules: 'true'
+
+      #
+      # Print system info
+      #
+
+      - name: System info
+        uses: ./openvino/.github/actions/system_info
+        
+      - name: Setup ccache
+        id: ccache_restore
+        uses: ./openvino/.github/actions/cache
+        with:
+          save-always: ${{ github.event_name == 'push' && 'true' || 'false' }}
+          cleanup-always: ${{ github.event_name == 'push' && 'true' || 'false' }}
+          cache-size: 10
+          max-cache-size: 50
+          cache-path: ${{ env.CCACHE_REMOTE_DIR }}
+          path: ${{ env.CCACHE_DIR }}
+          key: ${{ runner.os }}-${{ runner.arch }}-ccache-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-${{ runner.arch }}-ccache
+
+      #
+      # Build
+      #
+      - name: Clean ccache stats
+        run: ccache --zero-stats
+      
+      - name: CMake - Configure
+        run: >
+          cmake -G "${{ env.CMAKE_GENERATOR }}"
+          -DENABLE_CPPLINT=OFF
+          -DENABLE_INTEL_GPU=OFF
+          -DENABLE_INTEL_NPU=OFF
+          -DENABLE_SAMPLES=OF
+          -DENABLE_NCC_STYLE=OFF
+          -DENABLE_PYTHON=OFF
```

Comment:
Could you tell me please if you had a chance to validate build with Python API? I see that  Python API build is disabled at the moment.

Reply:
It is part of another task https://jira.devtools.intel.com/browse/CVS-159545